spec/integration/_dev/build/build.spec.yml

```diff
@@ -32,7 +32,7 @@ spec:
     - dependencies
 # JSON patches for newer versions should be placed on top
 versions:
-  - before: 3.2.0
+  - before: 3.1.3
```

Comment:
Revert change introduced in https://github.com/elastic/package-spec/pull/727/files#diff-4c8c0dc57af11ec06ff33e581abecb275af0b3418fa7daba98ed3e724572883aR35

Reply:
Good catch, thanks.